### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by wangping1408** [yinguobing/cnn-facial-landmark#87]
Hello Is there a new evaluation indicator? Is there only the root mean square error used in the verification, and is the accuracy of the NME evaluation test used?

**Comment by yinguobing** [comment]
I think this project needs a more formal way of evaluation. Could you please introduce a little more about NME?

**Comment by yinguobing** [comment]
Hi @wangping1408,

I have not found any matching built in metric from TensorFlow's[ official metrics](https://www.tensorflow.org/api_docs/python/tf/keras/metrics). Maybe we need to write it by ourselves.

There is a page about adding custom metrics in keras. But I'm not sure if TensorFlow support it. You can find it here: ](https://keras.io/metrics/#custom-metrics)
